appveyor.yml

```
@@ -1,23 +1,15 @@
 version: 4.0.0.{build}
 
-os: Visual Studio 2019
-
 branches:
   except:
     - configdata
     - gh-pages
 
-environment:
-  # The following entries define the default values of the environment variables.
-  # They can be overridden in AppVeyor | Settings | Environment.
-  matrix:
-  - IdeVersion: VS2019
-  SKIP_PAUSE: TRUE
-  ARCHIVE_WITH_PDB: TRUE
-  GE_TEST_SLEEP_SECONDS_ON_HANG: 0
-  GE_TEST_LAUNCH_DEBUGGER_ON_HANG: 0
-  TEST_RUN_COUNT: 1
-  IGNORE_TEST_RESULT: FALSE
+matrix:
+  fast_finish: true
+
+# https://www.appveyor.com/docs/build-environment/#build-worker-images
+image: Visual Studio 2019
```

The following two variables are evaluated by the current code, i.e. should be kept.

```suggestion
image: Visual Studio 2019

environment:
  # The following entries define the default values of the environment variables.
  # They can be overridden in AppVeyor | Settings | Environment.
  GE_TEST_SLEEP_SECONDS_ON_HANG: 0
  GE_TEST_LAUNCH_DEBUGGER_ON_HANG: 0
```